aip/cloud/2510.md

```diff
@@ -67,6 +67,46 @@ considered PII and user data, but project numbers are not.
 Therefore, when an internal service calls an external Google APIs, it
 **should** use project numbers for making API requests.
 
+### Resource References
+
+Project IDs also appear in [resource names][], which are stored in other
```

Comment:
I think, "...which are referred to by other...", would be more accurate as APIs are ephemeral and the whether things are *stored* is outside of the concern of an API surface.

Reply:
rephrased, PTAL.